staging/src/k8s.io/kube-aggregator/pkg/client/clientset_generated/clientset/typed/apiregistration/v1beta1/apiservice.go

```diff
@@ -57,6 +57,38 @@ func newAPIServices(c *ApiregistrationV1beta1Client) *aPIServices {
 	}
 }
 
+// Get takes name of the aPIService, and returns the corresponding aPIService object, and an error if there is any.
```

Comment:
Sorry, i didn't notice this file is in k8s.io/aggregator. Github hid the path. Please keep it.

Please only remove `k8s.io/kubernetes/pkg/client/clientset_generated/clientset`, if there is any.